show/main.py

```diff
@@ -966,6 +966,28 @@ def counters(ctx, verbose, period, printall):
 
         run_command(cmd, display_cmd=verbose)
 
+# 'errors' subcommand ("show interfaces counters errors")
+@counters.command()
+@click.option('-a', '--printall', is_flag=True)
+@click.option('-p', '--period')
+@click.option('--verbose', is_flag=True, help="Enable verbose output")
+@click.pass_context
+def errors(ctx, verbose, period, printall):
```

Comment:
Since the last commit, you're no longer using the context so you can remove line 974 and remove `ctx` from 975.

Also, in the last commit you removed the logic to handle the printall and period options. I don't know if printall makes sense with `errors` or `rate`, but the period feature is handy, and I could see use for it with these two subcommands.

Above comments also apply to `rate()` below

Reply:
Discussed offline. Deciding to omit the "period" option from "errors" and "rate" subcommands  for this PR. That functionality can be implemented at a later time.

Reply:
Thanks Joe.  I've updated the main.py file per our conversation.